tools/end2end_tests/scripts/clean_local.sh

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Would Docker Compose be better for this?

Reply:
+1